src/main/java/edu/kis/vh/nursery/containers/IntContainer.java

```diff
@@ -0,0 +1,19 @@
+package edu.kis.vh.nursery.containers;
+
+public interface IntContainer {
+
+    static final int empty = 0;
```

Comment:
PLEASE FIX: Constants should be written UPPERCASE.

Reply:
✔️